crypto-serialization/src/main/java/com/palantir/crypto/KeyMaterials.java

```diff
@@ -42,6 +44,15 @@
 
     private static final Logger log = LoggerFactory.getLogger(KeyMaterials.class);
     private static final Map<Integer, ? extends KeySerializer> SERIALIZERS = KeySerializers.getSerializers();
+    private static final CryptoRandom random = getRandom();
+
+    private static CryptoRandom getRandom() {
+        try {
+            return CryptoRandomFactory.getCryptoRandom();
```

Comment:
If the native crypto libs are available, and the hardware has the `RDRAND` instruction, it should use that. Otherwise, I gather from https://wiki.openssl.org/index.php/Random_Numbers that it will use `/dev/urandom` if the native libs are present. Without native libs, it'll just use a Java `SecureRandom`, presumably using whatever the default entropy source of the system is.

We could probably call `getCryptoRandom(properties)` where `properties` specifies the entropy source for Java to use. My understanding is that `/dev/urandom` is correct, and `/dev/random` is not (since it blocks)?